tomoj/protocols/__init__.py

```diff
@@ -25,3 +27,4 @@
 # **************************************************************************
 
 from .protocol_xCorrPrealignment import ProtTomojXcorrPrealignment
+from .protocol_landmarksGeneration import ProtTomojLandmarksGeneration
```

Comment:
I believe this line should not be there. ProtTomojLandmarksGeneration belongs to another brach.

Reply:
Yes it should not be there.